src/modules/session-recorder/utils.ts

```diff
@@ -0,0 +1,22 @@
+export const isMediaRequest = (contentType: string): boolean => {
+  const regexExp = /^(image|audio|video)\/.+$/gi;
+  return regexExp?.test(contentType);
+};
+
+export const getObjectSizeInBytes = (obj): number => {
+  if (!obj) {
+    return NaN;
+  }
+
+  let stringifiedObj: string = obj;
+  try {
+    if (typeof obj !== 'string') {
+      stringifiedObj = JSON.stringify(obj);
+    }
+  } catch (e) {
+    // skip
+  }
+
+  const size = new TextEncoder().encode(stringifiedObj).length;
```

Comment:
Do we see much difference if we directly use `stringifiedObj.length`?
If not, let's skip text encoding as it will not be efficient to do for each network requests and responses.